### PR TITLE
ghostscript 10.03.1

### PR DESCRIPTION
**Pull request by chenrui333**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

https://github.com/Homebrew/homebrew-core/actions/runs/9153023096/job/25161337314

```
==> Downloading https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs10030/ghostpdl-10.03.1.tar.xz
curl: (22) The requested URL returned error: 404
```

### CODE REVIEW

File: Formula/g/ghostscript.rb [hunk 1]
```diff
@@ -71,7 +70,7 @@ class Ghostscript < Formula
 
   def install
     # Delete local vendored sources so build uses system dependencies
-    libs = %w[expat freetype jbig2dec jpeg lcms2mt libpng openjpeg tiff zlib]
+    libs = %w[freetype jbig2dec jpeg lcms2mt libpng openjpeg tiff zlib]
```
Comment (carlocab) [inline Formula/g/ghostscript.rb:73]:
```suggestion
    libs = %w[expat freetype jbig2dec jpeg lcms2mt libpng openjpeg tiff zlib]
```
Let's not do this.

Comment (chenrui333) [reply Formula/g/ghostscript.rb:73]: yeah, the artifact was wrong, that is why, reverting this line.

File: Formula/g/ghostscript.rb [hunk 2]
```diff
@@ -4,8 +4,8 @@ class Ghostscript < Formula
   license "AGPL-3.0-or-later"
 
   stable do
-    url "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs10030/ghostpdl-10.03.0.tar.xz"
-    sha256 "854fd1958711b9b5108c052a6d552b906f1e3ebf3262763febf347d77618639d"
+    url "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs10031/ghostscript-10.03.1.tar.xz"
```
Comment (chenrui333) [inline Formula/g/ghostscript.rb:7]:
```suggestion
    url "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs10031/ghostpdl-10.03.1.tar.xz"
```

**Review by carlocab — APPROVED** [review]

**Comment by chenrui333** [comment]
two runs:
- https://github.com/Homebrew/homebrew-core/actions/runs/9159192780/job/25190384348
- https://github.com/Homebrew/homebrew-core/actions/runs/9159192780/job/25201484920

**Pull request merged**